app/scripts/controllers/transactions.js

```diff
@@ -413,18 +413,23 @@ export function TransactionsCtrl ($scope, $uibModal, $location, $timeout, $inter
 
   const refreshSuccess = function (transactions) {
     // on success
-    $scope.transactions = transactions
+    if (transactions.count) {
```

Comment:
```suggestion
    if (transactions.length) {
```
Is transactions a object or an array ?

Reply:
The endpoint can return an array of transactions or an object containing the count property, which is the length of the transactions to be rerun (for bulk reruns).

Reply:
Thanks for the clarification